bootstrap/README.md

```diff
@@ -11,17 +11,22 @@ comments to the Moderator database.
 
 To generate a JWT token for a service user:
 
-* Set the `TOKEN_SECRET` and `TOKEN_ISSUER` environment variables to the value
-they are set to in the instance of Moderator you are using. They will either be
-set as environment variables in the deployment process, or set to the default values
-in the Moderator [config](../packages/config/index.js).
-* Run the following command from the `/packages/cli` directory.
+* Set up the `TOKEN_SECRET` (and `TOKEN_ISSUER`) configuration items as described
+in the [root README](../README.md)
 
+* Create a service user (if you haven't already got one) and remember his USER_ID:
+
+```bash
+bin/osmod users:create --group service --name "Robot"
 ```
-./bin/osmod.js users:get-token --id={USER_ID}
+
+* Get the JW token for that user:
+
+```bash
+bin/osmod users:get-token --id={USER_ID}
```

Comment:
Here, USER_ID is Robot? Name use {USER_ID} consistently or "Robot" consistently, rather than mix them?